.github/workflows/build-electron.yml

```diff
@@ -51,7 +51,6 @@ jobs:
 
     - name: Test electron app with puppeteer
       uses: GabrielBB/xvfb-action@v1
```

Comment:
I'm pretty sure this xvfb is only used on linux systems, so we can probably drop this too.

Reply:
Yep, I just looked into this action and it just executes the command exactly as stated _unless_ it's on linux.  If on linux it'll use xvfb to simulate a display.

Reply:
Oh, great!